### DISCUSSION TITLE
Welcome to city-cab Discussions!

### DISCUSSION THREAD

**Discussion by edemekong** [hello-paulvin/city-cab#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edemekong** [comment]
Hello Everyone!👋

**Reply by BariCeo** [reply]
Hello sir
thanx for your work
can you please update bloc to the new version it will be verry helpfull

**Comment by BariCeo** [comment]
can you please update bloc to the new version?
 it's hard to use the app sometimes impossible

**Reply by edemekong** [reply]
Hey @BariCeo the app was rewritten with provider for more broader audience... 
To update this I will need sometime, to make this happen. Thanks
Check https://github.com/hello-paulvin/city-cab/tree/develop-provider for all the latest code on provider implementation.

**Comment by BariCeo** [comment]
Thanks a lot sir

**Comment by tizibienvenus** [comment]
Hello Edemekong

**Comment by tizibienvenus** [comment]
I want to discuss with you.

**Reply by edemekong** [reply]
Hey bro hit me up on Twitter(X) @edeme_kong

**Comment by tizibienvenus** [comment]
Hello How are you Paul ?

Le lun. 11 sept. 2023 à 13:38, Paul Jeremiah ***@***.***> a
écrit :
